lesson_6_2/homework_6_2.py

```diff
@@ -0,0 +1,11 @@
+if_h_found = False
+while(True):
+    entered_string = input("Please enter string which include 'h' or 'H' letter:")
+    entered_string = entered_string.upper()
+    list_of_characters = entered_string[::1]
```

Comment:
There is no need to convert str in list in this case. You can use count() on str.